pkg/kubelet/apis/deviceplugin/v1beta1/api.pb.go

```diff
@@ -361,6 +361,8 @@ type DeviceSpec struct {
 	// * w - allows container to write to the specified device.
 	// * m - allows container to create device files that do not yet exist.
 	Permissions string `protobuf:"bytes,3,opt,name=permissions,proto3" json:"permissions,omitempty"`
+	// id to identify the specific device
+	DeviceId string `protobuf:"bytes,4,opt,name=device_id,json=deviceId,proto3" json:"device_id,omitempty"`
```

Comment:
field name should be `DeviceID` with json of `deviceID`, right?

Reply:
+1